### DISCUSSION TITLE
How to let the search box only search the current page?

### DISCUSSION THREAD

**Discussion by elllllllaaa** [microsoft-search/pnp-modern-search#2952] [Q&A]
Hi, really hope someone could help me‌‌ been trying for many days. I am currently using SharePoint to create a page. 

I would like to create a search box for only search the current page, it's a bit like searching keywords which leads you to that place. Example here (but it's not on SharePoint):

![image](https://user-images.githubusercontent.com/132588823/236243065-06de7eb3-edb9-4ce5-84df-3110ef9cf382.png)

However, the solutions I found it's always link to other sites or pages.
![image](https://user-images.githubusercontent.com/132588823/236243199-4399743d-0cda-4db7-997f-7c3219d113e4.png)

But I only want the results to be current page. Does anyone know how to do this? Thank you so much!

**Comment by wobba** [comment]
You cannot use this solution to search on page. The search box has to be tied to a result web part. Not aware of a web part which does what you want.
